distributionviewer/core/static/js/app/components/containers/chart-list-container.js

```diff
@@ -1,25 +1,24 @@
 import React from 'react';
 import ChartList from '../views/chart-list';
 
-// numCharts is updated and provided to ChartList just for demonstration
-// purposes, and so that ChartList doesn't just have to include <ExampleChart />
-// 30 times in its source.
-const ChartListContainer = React.createClass({
-  getInitialState: function() {
-    return {
-      numCharts: 0
-    }
-  },
+export class ChartListContainer extends React.Component {
```

Comment:
This seems a little more verbose but it does make certain things like the constructor a bit more clear. Consistency will be important I guess.